[DQL3-54] Multiple fixes

tino097 opened the pull request:
[DQL3-54] Move the nature_of_change under url field

[DQL3-60] dont recalculate date when creating dataset with resource

[DQL3-58] dont validate next_update_due if thre is no changes in next_update_due and upate_frequency fields

tino097 committed:
[DQL3-54] Move the nature_of_change under url field

tino097 committed:
[DQL3-60] dont recalculate date when creating dataset with resource

tino097 committed:
[DQL3-58] dont validate next_update_due if thre is no changes in next_update_due and upate_frequency fields

MarkCalvert commented on ckanext/data_qld/resource_freshness/validation.py:
```diff
@@ -26,13 +26,19 @@ def validate_next_update_due(keys, flattened_data, errors, context):
 
+    pkg = data_qld_helpers.get_old_data(data, context)
+    
+    import pdb; pdb.set_trace()
```
Hey @tino097 you forgot to remove the pdb

MarkCalvert commented on ckanext/data_qld/helpers.py:
```diff
@@ -188,3 +188,7 @@ def is_delete_request():
+
+
+def get_old_data(data, context):
+    return toolkit.get_action('package_show')(context, {"id": data.get('id')})
```
You can get the package object from the `context.get('package')` and fall back to `package_show` if for some reason it is not there

MarkCalvert committed:
[DQL3-54]
- Refactored code to use 'context.get('package')' first to save a trip to solr to retreive the data from 'package_show'
